SPECS/prebuilt-ca-certificates/prebuilt-ca-certificates.spec

```diff
@@ -49,6 +49,9 @@ find %{buildroot} -name README -delete
 %{_sysconfdir}/pki/java/cacerts
 
 %changelog
+* Fri Jan 19 2024 Andrew Phelps <anphel@microsoft.com> - 3.0.0-2
+- Use asciidoc binary instead of asciidoc.py
```

Comment:
In case of `prebuilt-*` spec changes, comments like "- Making 'Release' match with 'ca-certificates'." below may be more accurate.

Reply:
sure, updated